src/cli/onefuzz/cli.py

```diff
@@ -490,18 +490,33 @@ def print_nested_help(self, args: argparse.Namespace) -> None:
             level += 1
 
 
+def render(result: Any) -> Any:
```

Comment:
So am I correct in reading the intent of this as: walk the argument, and turn it into something that is similar to what you'd get from a call to  `json.loads()`?

Reply:
Yes.  The previous implementation was extremely naive.  It couldn't handle the results as provided by this instance, specifically `Dict[UUID, int]`.

Reply:
Cool. Maybe just add a short comment emphasizing the sort of thing we're trying to build, for posterity? Based only on function name, I'd expect `render()` to return a string, when the real meaning here is more something like `as_json_like_dict()`.

Reply:
I renamed it to normalize.  It's not always a dict in response.  As an example, it can take a list of UUIDs and return a list of strings.